### PR TITLE
[ObjCRuntime/NSObject] Remove dead Classic code.

### PR DESCRIPTION
**Pull request by rolfbjarne**

### CODE REVIEW

File: src/ObjCRuntime/Runtime.cs [hunk 1]
```diff
@@ -835,11 +829,6 @@ internal static ProtocolMemberAttribute GetProtocolMemberAttribute (Type type, s
 		// delegate
 		//
 		[EditorBrowsable (EditorBrowsableState.Never)]
-#if XAMCORE_2_0
-		internal
-#else
-		public 
-#endif
 		static MethodInfo GetBlockWrapperCreator (MethodInfo method, int parameter)
```
Comment (Therzok) [inline src/ObjCRuntime/Runtime.cs:832]: Should be internal?

Comment (rolfbjarne) [reply src/ObjCRuntime/Runtime.cs:832]: Looks like it doesn't need to be (anymore?), so I just made it private.

**Comment by rolfbjarne** [comment]
build

**Comment by mandel-macaque** [comment]
build

**Pull request merged**